cli/command/service/scale.go

```diff
@@ -58,12 +65,11 @@ func runScale(dockerCli *command.DockerCli, args []string) error {
 	return fmt.Errorf(strings.Join(errors, "\n"))
 }
 
-func runServiceScale(dockerCli *command.DockerCli, serviceID string, scale string) error {
+func runServiceScale(dockerCli *command.DockerCli, serviceID string, uintScale uint64) error {
```

Comment:
I don't think the variable needs to be renamed to include the type, just `scale` should do